tools/package-tools/compile-entry-point.ts

```diff
@@ -10,13 +10,19 @@ let nextId = 0;
 
 /** Compiles the TypeScript sources of a primary or secondary entry point. */
 export async function compileEntryPoint(buildPackage: BuildPackage, tsconfigName: string,
-                                          secondaryEntryPoint = '') {
+                                          secondaryEntryPoint = '', es5OutputPath?: string) {
   const entryPointPath = join(buildPackage.sourceDir, secondaryEntryPoint);
   const entryPointTsconfigPath = join(entryPointPath, tsconfigName);
+  const ngcFlags = ['-p', entryPointTsconfigPath];
+
+  if (es5OutputPath) {
+    ngcFlags.push('--outDir', es5OutputPath);
```

Comment:
You can combine these into a single push: `ngcFlags.push('--outDir', es5OutputPath, '--target', 'ES5')`.

Reply:
True, I just wanted to keep it separate since those are two different option-value pairs and this way it feels more readable.